tools/build/options/WITHOUT_CHERI

```diff
@@ -1,2 +1,2 @@
 .\" $FreeBSD$
-Set to not build software requiring CHERI capability support.
+Do not build software requiring CHERI capability support.
```

Comment:
```suggestion
Do not build optional software requiring CHERI capability support.
```
This mostly controls little test bits like cheribsdtest. (I've got a patch I'll submit shortly to make it only do this).